lib/mechanize.rb

```diff
@@ -1299,6 +1299,13 @@ def shutdown
     @agent.shutdown
   end
 
+  ##
+  # Allow Query Parameter values to be empty
+
+  def allow_empty_query_values allow_empty
+    @agent.allow_empty_query_values = allow_empty
+  end
```

Comment:
This code isn't under test. Also, it feels awkward. Why isn't it written as a setter?